### DISCUSSION TITLE
Should the GPS Data be editable?

### DISCUSSION THREAD

**Discussion by d3liaa** [noi-techpark/it.bz.opendatahub.databrowser#311] [Q&A]
Should the GPS Data be editable? Given this design [https://prnt.sc/ojoc37IvI6By](https://prnt.sc/ojoc37IvI6By)
If yes how should we do it? Will it use the new map component to choose from there the new coordinates?

**Comment by sseppi** [comment]
Hi @d3liaa ,

in my opinion the GPS Data should be editable and in a first iteration I would suggest to not use a map but only let the user imput the right numbers for Latitude and Longitude.

In a second iteration, we could extend the component by adding a map that allows to visulize, put ore move the pins on a map.

What do the others think?

**Comment by gappc** [comment]
@sseppi for me that's fine, In addition I would make the altitude information editable, too.

**Comment by sseppi** [comment]
@gappc fine for me, we decided that all fields but ID that are editable on API side are also editable in the Data Browser.
